website/docs/exempt-namespaces.md

```diff
@@ -5,6 +5,8 @@ title: Exempting Namespaces
 
 ## Exempting Namespaces from Gatekeeper using config resource
 
+> The "Config" resource has to be named "config" for it to be reconciled by Gatekeeper. Gatekeeper will ignore the resource if you do not name it "config".
```

Comment:
```suggestion
> The "Config" resource must be named `config` for it to be reconciled by Gatekeeper. Gatekeeper will ignore the resource if you do not name it `config`.
```

Reply:
I have updated the `sync.md` as well to reflect the stronger wording.